src/content/changes/highlights/2022-09-30.md

```diff
@@ -0,0 +1,14 @@
+---
+date: 2022-09-30T10:00:00
+title: Highlights for the week ending September 30, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## User interfaces
+
+## Apps
+
```

Comment:
```suggestion
- [fluent-logshipping-app](https://github.com/giantswarm/fluent-logshipping-app) version [v2.0.0](https://github.com/giantswarm/fluent-logshipping-app/blob/master/CHANGELOG.md#200---2022-09-28) This update supports CRI instead of the docker Format and changes the storage path to not use a tmpfs to not overload the nodes
```